SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -21,18 +22,33 @@ public DefaultApplicationSettings()
         {
             lock (configLock)
             {
-                configuration = OpenConfiguration();
+                try
+                {
+                    configuration = OpenConfiguration();
+                }
+                catch (Exception e)
+                {
+                    AppCenterLog.Error(AppCenterLog.LogTag, "Configuration file could be corrupted.", e);
+                    RestoreConfigurationFile();
+                }
             }
         }
 
         public T GetValue<T>(string key, T defaultValue = default(T))
         {
             lock (configLock)
             {
-                var value = configuration.AppSettings.Settings[key];
-                if (value != null)
+                if (configuration != null)
+                {
+                    var value = configuration.AppSettings.Settings[key];
+                    if (value != null)
+                    {
+                        return (T)TypeDescriptor.GetConverter(typeof(T)).ConvertFromInvariantString(value.Value);
```

Comment:
I know that's not your piece of code here, but consider changing name `value`, to avoid having `value.Value` in code :)

Reply:
Fixed